prac_08/silver_service_taxi.py

```diff
@@ -1,20 +0,0 @@
-"""
-CP1404/CP5632 Practical
-SilverServiceTaxi class
-"""
-from prac_08.taxi import Taxi
```

Comment:
the taxi.py code imports this SilverServiceTaxi class (not the other way around).... I don't think you need this line of code?... but again I haven't done the prac yet so take this comment with a grain of salt